public/csv/car_report_1735133202.csv

```diff
@@ -0,0 +1,42 @@
+"Brand and Model",Count
```

Comment:
you still have this file, delete it. When commiting make sure to check which files you commit. Once you remove it, add in git ignore public/csv so it will ignore files in that dir